api/python/quilt3/main.py

```diff
@@ -149,7 +150,17 @@ def cmd_catalog(navigation_target=None, detailed_help=False):
 
     If detailed_help=True, display detailed information about the `quilt3 catalog` command and then exit
     """
-    from .registry import app  # Delay importing it cause it's expensive.
+    try:
+        from .registry import app  # Delay importing it cause it's expensive.
+    except ModuleNotFoundError as e:
+        if e.name not in (
+            'flask',
+            'flask_cors',
+            'flask_json',
+        ):
+            raise
+        print('Running `quilt3 catalog` requires you to install quilt3[catalog]')
```

Comment:
```suggestion
        print(To use `quilt3 catalog` you must first install quilt3[catalog]')
```

Reply:
I adjusted message, but removed `optional` since I think it's quite obvious :slightly_smiling_face: .